src/Module.php

```diff
@@ -102,6 +102,12 @@ class Module extends \yii\base\Module implements BootstrapInterface
      * @since 2.1.1
      */
     public $defaultHeight = 50;
+    /**
+     * @var string toolbar position on web page. Use 'bottom' or 'upper'.
```

Comment:
`top` is cleaner than `upper`. Let's use it instead.

Reply:
`yii-debug-toolbar_position_top` css class is already in use inside `index.php` and `view.php` templates (I miss it earlier).
Should I rename the old one (to `yii-debug-toolbar_position_top_expanded`)?

Reply:
Is old one used?

Reply:
Yes, in two places.